src/tables.ts

```diff
@@ -332,6 +332,18 @@ export const Tables = {
    * For example: Vex.Flow.DEFAULT_FONT_STACK = [Fonts.Petaluma(), Fonts.Custom()];
    */
   DEFAULT_FONT_STACK: [Fonts.Bravura(), Fonts.Gonville(), Fonts.Custom()],
+
+  /**
+   * @returns the `Font` object at the head of the music font stack.
+   */
+  currentMusicFont(): Font {
+    if (Tables.DEFAULT_FONT_STACK.length === 0) {
+      throw new RuntimeError('NoFonts', 'The font stack is empty. Call: Flow.setMusicFont(...fontNames)');
+    } else {
+      return Tables.DEFAULT_FONT_STACK[0];
```

Comment:
Hopefully this will rebase cleanly. :-)

In the final version, this will be named `Tables.MUSIC_FONT_STACK[0];`.

I really wanted to make it clear to anyone reading the code the difference between:
1) fonts for rendering music symbols
2) fonts for displaying text like lyrics or annotations

That's why I prefer calling this MUSIC_FONT_STACK.

Reply:
I also think that it will rebase fine. No change required.